src/Link.tsx

```diff
@@ -1,44 +1,65 @@
+import {LocationDescriptor} from 'history';
```

Comment:
Nitpick: I know I didn't configure a linter but please add spaces inside the braces. 🙈